tensorflow/lite/kernels/kernel_util_test.cc

```diff
@@ -14,11 +14,11 @@ limitations under the License.
 ==============================================================================*/
 #include "tensorflow/lite/kernels/kernel_util.h"
 
-#include <math.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 
+#include <cmath>
```

Comment:
why is this needed?

Reply:
that's an interesting / weird case, when building with certain NDK, I saw some non-ANSI/ISO libm function not declared in `<math.h>` but in `<cmath>. I'll check if I can find the case. Otherwise, I'll revert it.

Reply:
it was a `std:ldexp()` problem on when building with NDK r20 on Linux at that time. It's no longer a problem in recent master branch.